### PR TITLE
[Fix #1281] Add auto-correct for Style/IfUnlessModifier cop

### LINKED ISSUE #1281: Add autocorrect for Style/IfUnlessModifier cop
References https://github.com/bbatsov/rubocop/issues/743.

### PR DESCRIPTION
**Pull request by lumeet**

### CODE REVIEW

File: spec/rubocop/cop/style/if_unless_modifier_spec.rb [hunk 1]
```diff
@@ -11,39 +11,70 @@
...
 
-  it 'registers an offense for multiline if that fits on one line' do
-    # This if statement fits exactly on one line if written as a
-    # modifier.
-    condition = 'a' * 38
-    body = 'b' * 36
-    expect("  #{body} if #{condition}".length).to eq(80)
+  context 'multiline if that fits on one line' do
+    let(:source) do
+      ["  if #{condition}",
```
Comment (bbatsov) [inline spec/rubocop/cop/style/if_unless_modifier_spec.rb]: Let's trim this leading whitespace.

Comment (lumeet) [inline spec/rubocop/cop/style/if_unless_modifier_spec.rb]: @bbtatsov Do you mean just cleaning up the spec or is there something else I can't see? :)

Comment (bbatsov) [inline spec/rubocop/cop/style/if_unless_modifier_spec.rb]: There's some leading space inside the strings. Seems totally redundant to me.

**Comment by bbatsov** [comment]
:+1: Thanks!

**Pull request merged**